.eleventy.js

```diff
@@ -87,41 +72,37 @@ module.exports = function(eleventyConfig) {
 	});
 
 	eleventyConfig.addCollection("allTags", collection => {
-		const tags = getUniqueTags(collection.getAll());
+		let postsByTag = [];
+
+		getUniqueTags(collection.getAll()).forEach(tag => {
+			postsByTag.push({ ... tag, posts: getFilteredByTagTransformed(tag.slug, collection, { transform: slugFilter }) });
```

Comment:
It seems `getFilteredByTagTransformed` is only used once at here. I would suggest to remove the 3rd `options` argument and directly define and use `slugFilter` in `get-filtered-by-tag-transformed.js`. This will make it easier to read the entire handling in one file rather than coming back here to check what's the transform function.

By doing so, the definition of `slugFilter` at line 19 can be removed from this file.

When the time comes that `getFilteredByTagTransformed` may called more with various transform functions, we could do such refactoring.